firmware/quadruna/BMS/src/app/app_heartbeatMonitor.c

```diff
@@ -0,0 +1,53 @@
+#include "app_heartbeatMonitor.h"
+#include "app_heartbeatMonitorBoard.h"
+
+#include "app_canRx.h"
+#include "app_canTx.h"
+#include "app_canAlerts.h"
+
+HeartbeatMonitorBoard vc_hbmonitor = {
```

Comment:
Nit: This should probably be named something more consistent with the naming scheme on the rest of the codebase (`VC_heartbeat_monitor` or `VC_hb_monitor`?)